golang/cosmos/x/vbank/client/rest/rest.go

```diff
@@ -0,0 +1,57 @@
+package rest
```

Comment:
Consider writing tests for this package, or make a TODO/issue to do so later.

Reply:
I tried to port the tests from Cosmos-SDK, but all the client tests are intertwined with the "simapp", which is incompatible with externally-defined modules like what we have.

So yes, yuck.

Reply:
Deleted.